tests/upgrade/rules/test_base_rule.py

```diff
@@ -25,3 +25,20 @@ def test_rules_are_registered(self):
         rule_classes = get_rules()
         assert BaseRule not in rule_classes
         assert ConnTypeIsNotNullableRule in rule_classes
+
+    def test_rules_are_ordered(self):
+        rule_classes = get_rules()
+        from airflow.upgrade.rules.aaa_airflow_version_check import VersionCheckRule
+        from airflow.upgrade.rules.conn_id_is_unique import UniqueConnIdRule
+        from airflow.upgrade.rules.no_additional_args_in_operators import (
+            NoAdditionalArgsInOperatorsRule,
+        )
+
+        # Version check should still be first
+        assert rule_classes[0] == VersionCheckRule
+        print(rule_classes)
```

Comment:
```suggestion
```

Reply:
GAH thank you, left in my debugging print 🙃